Gemfile

```diff
@@ -61,3 +63,5 @@ gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 
 # JQuery 
 gem 'jquery-rails'
+
+gem "rspec-rails", "~> 3.5.0"
```

Comment:
The gem's [README](https://github.com/rspec/rspec-rails#installation) suggests including the gem inside the `development` and `test` groups of the Gemfile.

Also, it mentions restricting the version to `'~> 3.8'`. Did you encounter any issues which led you to declare this specific version restriction?